### ISSUE TITLE
Visualization: linear viz doesn't properly show binary images

### ISSUE THREAD

**Issue by sk1p** [LiberTEM/LiberTEM#944]
LiberTEM version: 0.5.1 or later (didn't test earlier)

Test code:

```python
from libertem.viz import visualize_simple
import matplotlib.pyplot as plt
import numpy as np

my_choice = np.random.choice(size=(128, 128), a=[0, 1], p=[0.8, 0.2])
rgb_data = visualize_simple(my_choice)
plt.figure()
plt.imshow(rgb_data)
```

Should show some sparse 10% random 1/0 noise, all-black instead.

Probably a one-off error, compare with result of this:

```python
my_choice = np.random.choice(size=(128, 128), a=[0, 1, 2], p=[0.8, 0.1, 0.1])
rgb_data = visualize_simple(my_choice)
plt.figure()
plt.imshow(rgb_data)
```

Another way to reproduce: point analysis on 1bit data set.

**Comment by sk1p** [comment]
Will hopefully be fixed in #1011

**Comment by uellue** [comment]
Fix confirmed with a boolean test dataset
